doc/tutorial.rst

```diff
@@ -22,11 +22,12 @@ order to identify a particular IO-device if you have many devices.
 
 .. image:: illustrations/TutorialOverview.png
 
-The "data LED" on the IO-device is controlled by the IO-controller (PLC), and
-is normally flashing. With button 1 on the IO-device it is possible to tell
-the PLC to turn on and off the flashing of the LED.
+The LED1 ("data LED") on the IO-device is controlled by the IO-controller
+(PLC), and is normally flashing.  With Button1 on the IO-device it is
```

Comment:
Inconsequent use of "Button" and "button" (use Button in table above)

Reply:
Done